docs/supported_metadata_files.md

```diff
@@ -78,3 +78,27 @@ Will be interpreted as:
   "last_name": "Smith"
 }
 
+
+## Supported Metadata Files in SOMEF
+
+SOMEF can extract metadata from a wide range of files commonly found in software repositories. Below is a list of supported file types, along with clickable examples from real projects:
+
+| File Name          | Language       | Description | Detail | Source Spec. | Version Spec.| Example | Based  |
+|--------------------|----------------------------|-------------|--------|--------|---------------|-----------|----------|
+| `AUTHORS.md`       | General                    | Lists contributors, authors, and affiliations relevant to the project |  <div align="center">[🔍](./author.md)</div>| [📄](https://opensource.google/documentation/reference/releasing/authors/)| |[Example](https://gist.github.com/juliengdt/91d80c812e41be891dcf) |
```

Comment:
Group above all the things that are `General`. Then go with the languages in alphabetical order.